src/views/ManageList.jsx

```diff
@@ -55,6 +59,20 @@ export function ManageList() {
 		[itemName, daysUntilNextPurchase, listPath],
 	);
 
+	const handleEmailInputSubmit = async (event) => {
```

Comment:
The integration between `shareList` and `handleEmailInputSubmit` looks solid! Nicely done! If you consider changing `shareList` to always return a consistent response, just be sure to update the handling in `handleEmailInputSubmit` as well to keep everything aligned.